webcompat/templates/home-page/form.html

```diff
@@ -69,7 +69,7 @@ <h1 class="headline-1">Report Site Issue</h1>
     </div>
     <div class="form-text form-element js-Form-group">
       {{ form.os(class_='form-field text-field') }}
-      <span class="form-input-validation check js-bug-form-os"></span>
+      <span class="form-input-validation check"></span>
```

Comment:
We need this class, it was added here to fix a failing test: https://github.com/webcompat/webcompat.com/commit/d07e1bb5409635abc3cb07cdf234a7ac6af82422

Reply:
(i'm adding this)